Makefile.base

```diff
@@ -14,7 +14,8 @@ endif
 unexport DIRS
 DIRS := $(sort $(abspath $(DIRS)))
 
-MODULE ?= $(shell basename $(CURDIR))
+_MOD := $(shell basename $(CURDIR))
+MODULE ?= $(_MOD)
```

Comment:
```sh
make SHELL='/bin/sh -x' 2>&1 | less
```
does indeed show a huge number of superfluous shell-outs to basename. Good catch!